var/spack/repos/builtin/packages/darshan-util/package.py

```diff
@@ -31,6 +31,8 @@ class DarshanUtil(Package):
     depends_on('zlib')
     depends_on('bzip2', when="+bzip2", type=("build", "link", "run"))
 
+    patch('retvoid.patch')
```

Comment:
In that case you want something like this:
```suggestion
    patch('retvoid.patch', when='@:3.2.1')
```

Reply:
Thank you for your kind suggestion.

Reply:
This sounds good to me, especially with the version qualifier on the patch.

Reply:
This return value bug should only affect versions 3.2.0 and 3.2.1, and we'll have it fixed in the next release. So, probably just `when='@3.2.0:3.2.1'` -- I imagine the patch would fail on previous versions.

Otherwise, looks good to me! Thanks!